src/GraphQL.Serialization.Newtonsoft/SchemaExtensions.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Threading.Tasks;
+using GraphQL.Types;
+
+namespace GraphQL.Serialization.Newtonsoft
+{
+    public static class SchemaExtensions
+    {
+        public static Task<string> ExecuteAsync(this ISchema schema, Action<ExecutionOptions> configure)
```

Comment:
I do not think this method will have consumers.

Reply:
Its used in unit tests and is part of the public API currently.  If users upgrade to 3.0 and add a reference to GraphQL.Serialization.Newtonsoft, their code can at least keep working as before.

Reply:
This method is specifically for new users "getting started" with the library, so they don't have to do a ton of setup just to run a query and see the results.

Reply:
This method is specifically for new users "getting started" with the library, so they don't have to do a ton of setup just to run a query and see the results.

Reply:
This method has little value for the calling code. It can be moved to test project.
Yes, it is a part of oublic API, but we have already changed it a lot working on 3.0.0 version. Also, the reduction of public api itself can be considered as a positive feature.

Reply:
> This method is specifically for new users "getting started"

If so, let it stay.

Reply:
@sungam3r It has a ton of value and should absolutely stay.  There needs to be an easy "let me try this out" without needing to new up several classes.  That was a major point of feedback for when this library was first introduced.

Reply:
>  That was a major point of feedback for when this library was first introduced.

Well, I don’t know much.

Reply:
np - there was more than one person that mentioned it, but that feedback also came directly from Lee Byron.